From 4e1d9c31ce6dab979e291b0ccdd1aab08fc0a4e8 Mon Sep 17 00:00:00 2001
From: Mario Haustein <mario.haustein@hrz.tu-chemnitz.de>
Date: Sat, 23 Aug 2025 12:45:40 +0200
Upstream: https://github.com/indilib/indi-3rdparty/pull/1148
Subject: [PATCH 1/2] Fix detection of system libusbp-1

Its *.pc file is installed as `libusbp-1` instead of `libusbp`.
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -41,7 +41,7 @@ message(STATUS "Found Libusb-1.0 via pkg-config: Includes=${LIBUSB1_PC_INCLUDE_D
 target_include_directories(indi_ticfocuser-ng PUBLIC ${LIBUSB1_PC_INCLUDE_DIRS})
 target_link_libraries(indi_ticfocuser-ng ${LIBUSB1_PC_LIBRARIES})
 
-pkg_check_modules(LIBUSBP_PC libusbp)
+pkg_check_modules(LIBUSBP_PC libusbp-1)
 
 if (NOT LIBUSBP_PC_FOUND)
     message(STATUS "LIBUSBP-1 (libusbp) not found on system via pkg-config. Trying to build from bundled libusbp.")
-- 
2.49.1

From f65d98ac7a3853f61b6bdffbd3c5e118bad70e30 Mon Sep 17 00:00:00 2001
From: Mario Haustein <mario.haustein@hrz.tu-chemnitz.de>
Date: Sat, 23 Aug 2025 12:47:04 +0200
Subject: [PATCH 2/2] Fix typo

Missing letter `S` in `LIBTIC-1_INCLUDE_DIRS`
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -137,8 +137,8 @@ if(NOT WITH_LIBTIC)
             ${CMAKE_CURRENT_SOURCE_DIR}/connection/driver_interfaces/PololuUsbInterface.cpp
             ${CMAKE_CURRENT_SOURCE_DIR}/connection/PololuUsbConnection.cpp
         )
-        if(LIBTIC-1_INCLUDE_DIR)
-            target_include_directories(indi_ticfocuser-ng PUBLIC ${LIBTIC-1_INCLUDE_DIR})
+        if(LIBTIC-1_INCLUDE_DIRS)
+            target_include_directories(indi_ticfocuser-ng PUBLIC ${LIBTIC-1_INCLUDE_DIRS})
         endif()
         target_link_libraries(indi_ticfocuser-ng "${LIBTIC-1_LIBRARIES}")
         set(WITH_LIBTIC TRUE)
-- 
2.49.1

